dolphinscheduler-task-plugin/dolphinscheduler-task-http/src/main/java/org/apache/dolphinscheduler/plugin/task/http/HttpParameters.java

```diff
@@ -17,47 +17,37 @@
 
 package org.apache.dolphinscheduler.plugin.task.http;
 
-import org.apache.dolphinscheduler.plugin.task.api.model.ResourceInfo;
 import org.apache.dolphinscheduler.plugin.task.api.parameters.AbstractParameters;
 
 import org.apache.commons.lang3.StringUtils;
 
-import java.util.ArrayList;
 import java.util.List;
 
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * http parameter
  */
+@EqualsAndHashCode(callSuper = true)
+@Data
 public class HttpParameters extends AbstractParameters {
 
-    /**
-     * url
-     */
     private String url;
 
-    /**
-     * httpMethod
-     */
-    private HttpMethod httpMethod;
+    @JsonProperty("httpMethod")
+    private HttpRequestMethod httpRequestMethod;
 
-    /**
-     *  http params
-     */
-    private List<HttpProperty> httpParams;
+    @JsonProperty("httpParams")
+    private List<HttpProperty> httpRequestParams;
 
```

Comment:
It seems not necessary to change these field names.

Reply:
Yes, but I think a more specific name would be better understood. WDYT? @caishunfeng

Reply:
fine